src/main/java/se/sundsvall/businessrules/api/RuleEngineResource.java

```diff
@@ -35,11 +35,11 @@ public class RuleEngineResource {
 	private RuleEngineService ruleEngineService;
```

Comment:
Just a curious question, and unrelated to this PR per se, but should field injection and `ReflectionUtils#setField(...)` to accomplish the dependency injection in tests really be used?